test-result-summary-client/src/Build/PossibleIssues.jsx

```diff
@@ -29,12 +29,22 @@ export default class PossibleIssues extends Component {
         });
         if (response.ok) {
             const relatedIssues = await response.json();
-            let dataSource = [];
+            let dataSource = {};
+            const repoUrlAPIPrefix = "https://api.github.com/repos/";
             for (let index = 0; index < relatedIssues.items.length; index++) {
-                dataSource.push({
-                    key: index,
-                    issues: <a href={relatedIssues.items[index].html_url} target="_blank" rel="noopener noreferrer">{relatedIssues.items[index].title}</a> 
-                }); 
+                const repoName = relatedIssues.items[index].repository_url.replace(repoUrlAPIPrefix, "");
+                const issueData = <a href={relatedIssues.items[index].html_url} target="_blank" rel="noopener noreferrer">{relatedIssues.items[index].title}</a>;
+                if (repoName in dataSource) {
+                    dataSource[repoName].push({
+                        key: Object.keys(dataSource[repoName]).length,
+                        issues: issueData
+                    });
+                } else {
+                    dataSource[repoName] = [{
+                        key: 0,
+                        issues: issueData                       
+                    }];
+                }
```

Comment:
Thanks @LongyuZhang 

- To make this code simpler:
```
// rename issueData to issue
dataSource[repoName] = dataSource[repoName] || [];
dataSource[repoName].push({
   key: dataSource[repoName].length,
   issue  
});
```

- In the above code, we change `issues` to `issue`. Please update the `columns` to reflect this